cmd/lifecycle/analyzer.go

```diff
@@ -200,7 +200,7 @@ func (aa analyzeArgs) analyze() (platform.AnalyzedMetadata, error) {
 		Logger:                cmd.DefaultLogger,
 		Platform:              aa.platform,
 		Image:                 img,
-		LayerMetadataRestorer: lifecycle.NewLayerMetadataRestorer(cmd.DefaultLogger, aa.layersDir, aa.platform06.skipLayers),
+		LayerMetadataRestorer: lifecycle.NewLayerMetadataRestorer(cmd.DefaultLogger, aa.layersDir, aa.platform06.skipLayers, true),
```

Comment:
Regarding creating a variable, I didn't think it's necessary because when you use an IDE, it usually shows you what variable you're setting, but I added it.

Regarding the analyzer acceptance tests, I think that we're already testing that there are no sha files [here](https://github.com/buildpacks/lifecycle/blob/main/acceptance/analyzer_test.go#L1074-L1087). Do you want to add something else?

Reply:
> Regarding the analyzer acceptance tests, I think that we're already testing that there are no sha files here. Do you want to add something else?

I missed that. Great!

Reply:
Sorry, I should have said `restorer` acceptance tests! For newer platform api...